src/Catel.MVVM/Catel.MVVM.Shared/MVVM/Commands/Command.cs

```diff
@@ -244,7 +244,7 @@ public void Execute(object parameter)
         public void Execute(TExecuteParameter parameter)
         {
 #pragma warning disable 4014
-            ExecuteAsync(parameter, false);
+            ExecuteAsync(parameter, false).Wait();
```

Comment:
Correct. The point is that for async tasks, one should use TaskCommand (which is awaited). .Wait will block the calling thread.